nuitka/build/static_src/OnefileBootstrap.c

```diff
@@ -638,7 +654,34 @@ int main(int argc, char **argv) {
 #endif
 
 #if defined(_WIN32)
-    res = SetFilePointer(exe_file, -8, NULL, FILE_END);
+    /* if an application is signed, the signature is at the end of the file
+       where we normally expect the start position of out container.
+       the overcome this limitation, use the windows function MapAndLoad()
+       to parse the PE header. The header contains information whether
+       a signature is present and at which address the first signature
+       start. so we can use that address to find the start position value */
+    DWORD cert_table_addr = 0;
+
+    PSTR exe_filename_a = convertPcwstrToPstr(exe_filename);
+    if (exe_filename_a) {
+        LOADED_IMAGE loaded_image;
+        if (MapAndLoad(exe_filename_a, "\\dont-search-path", &loaded_image, false, true)) {
```

Comment:
This need to convert to ANSI path, is worrisome. I guess using `GetShortPathNameW` before would make sure it works.